pyaedt/generic/design_types.py

```diff
@@ -1750,45 +1750,63 @@ def Siwave(
 }
 
 
-def get_pyaedt_app(project_name=None, design_name=None):
+def get_pyaedt_app(project_name=None, design_name=None, desktop=None):
     """Returns the Pyaedt Object of specific project_name and design_name.
 
     Parameters
     ----------
-    project_name
-    design_name
+    project_name : str, optional
+        Project name.
+    design_name : str, optional
+        Design name.
+    desktop :  :class:`pyaedt.desktop.Desktop`, optional
+        Desktop class.
```

Comment:
```suggestion
    desktop : :class:`pyaedt.desktop.Desktop`, optional
        Desktop class. The default is ``None``.
```
What behavior occurs when the default of ``None`` is used. Should we indicate it in this format: "The default is ``None``, in which case ...."